Cargo.toml

```diff
@@ -121,6 +121,8 @@ arbitrary = { version = "1.2.3", features = ["derive"] }
 arc-swap = "1.5"
 assert_matches = "1.5.0"
 async-trait = "0.1.58"
+aurora-engine-transactions = { git = "https://github.com/aurora-is-near/aurora-engine.git", tag = "3.6.1" }
```

Comment:
Is there a reason these are a git dependency?

We should avoid git dependencies where possible. The fact that this is referencing a tag suggests to me that these should be on crates.io.

Reply:
Aurora doesn't have a regular process for releasing crates to crates.io. So for now a git dep is required to use the crate. But this is easy to update in the future if Aurora does start releasing crates.

Reply:
I think it should really be a not "if Aurora starts", but rather "integrating with the rest of the ecosystem ought to be a very strong incentive for Aurora to start releasing crates as they're expanding."

We can't really audit git tags, they are not reproducible and might just make the build process slower every time as cargo can't cache git repositories globally.

Does it block this PR? Probably not. But waiting for Aurora to realize these problems on their own before an action is taken is also not great.